torch/nn/utils/_expanded_weights/embedding_expanded_weights.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.nn.functional as F
+from .expanded_weights_impl import implements_per_sample_grads
+from .expanded_weights_utils import standard_kwargs, forward_helper, set_grad_sample_if_exists
+
+from typing import List, Optional
+
+@implements_per_sample_grads(F.embedding)
+class EmbeddingPerSampleGrad(torch.autograd.Function):
+    @staticmethod
+    def forward(ctx, kwarg_names, _, *expanded_args_and_kwargs):
+        expanded_args, expanded_kwargs = standard_kwargs(kwarg_names, expanded_args_and_kwargs)
```

Comment:
Nit: if I am understanding this correctly, the operators that don't standardize are the ones that go through the PyTorch dispatcher first. Is that right?

https://github.com/pytorch/pytorch/blob/fb47cff737ec5aa5f3451db5c877410e0ae18141/torch/nn/utils/_expanded_weights/expanded_weights_utils.py#L5-L6

Reply:
I think that it's the ones where the `handle_torch_function` is generated in the Python/C++ binding layer. I can't find the exact generation for this but I think it will look something like [this] (https://github.com/pytorch/pytorch/blob/40d1f77384672337bd7e734e32cb5fad298959bd/tools/autograd/templates/python_nn_functions.cpp#L46) where it just passes the args and kwargs through as PyObjects

So I think it hits the Python/C++ binding layer but not the PyTorch dispatcher (but I might be missing something)